### PR TITLE
Updated Infinity Bot List

### PR DESCRIPTION
**Pull request by TheRealToxicDev**
- Logo
- POST Endpoint

### CODE REVIEW

File: data/lists/infinitybotlist.com.json [hunk 1]
```diff
@@ -3,14 +3,14 @@
...
-    "icon": "https://i.imgur.com/ibzF3CE.png",
+    "icon": "https://cdn.infinitybots.xyz/images/png/Infin2.png",
     "language": "English",
     "display": 1,
     "defunct": 0,
     "discord_only": 1,
     "description": "Infinity Bot List, Search your favourite bot for an exciting start to your server.",
     "api_docs": "https://docs.infinitybotlist.com/",
-    "api_post": "https://api.infinitybotlist.com/bot/:id",
+    "api_post": "https://api.infinitybotlist.com/bots/stats",
```
Comment (MattIPv4) [inline data/lists/infinitybotlist.com.json:13]: BotBlock does not send the bot ID in the JSON body payload, so if `:id` is no longer present in the URL I'm unsure how you'll determine which bot a POSTed count is for from BotBlock?

Comment (TheRealToxicDev) [reply data/lists/infinitybotlist.com.json:13]: We use the Bots Auth Token via the HEADER in the Request 😁

Comment (MattIPv4) [reply data/lists/infinitybotlist.com.json:13]: Ah! Nice.

**Review by MattIPv4 — APPROVED** [review]
LGTM

**Pull request merged**